lib/run.go

```diff
@@ -200,6 +200,10 @@ func (a *ACBuild) renderACI(insecure, debug bool) ([]string, error) {
 	for _, dep := range man.Dependencies {
 		err := reg.FetchAndRender(dep.ImageName, dep.Labels, dep.Size)
 		if err != nil {
+			if err.Error() == "bad HTTP status code: 404" {
```

Comment:
Hmm .. yes, this is not nice indeed. How about the following change which I quickly coded: https://github.com/s-urbaniak/build/commit/c9095272f3d91b9fddfbfcf8ad0a123d205e8a97?
It seems we already have an `ErrNotFound` error value in `lib`, so we can leverage that.